fee/payload.go

```diff
@@ -4,69 +4,26 @@ import (
 	"math/big"
 )
 
-const (
-	baseTxFee            = "2000000000000000" // 0.002 AERGO
-	payloadMaxSize       = 200 * 1024
-	StateDbMaxUpdateSize = payloadMaxSize
-	freeByteSize         = 200
-)
-
-var (
-	baseTxAergo   *big.Int
-	zeroFee       bool
-	stateDbMaxFee *big.Int
-	aerPerByte    *big.Int
-)
-
-func init() {
-	baseTxAergo, _ = new(big.Int).SetString(baseTxFee, 10)
-	zeroFee = false
-	aerPerByte = big.NewInt(5000000000000) // 5,000 GAER, feePerBytes * PayloadMaxBytes = 1 AERGO
-	stateDbMaxFee = new(big.Int).Mul(aerPerByte, big.NewInt(StateDbMaxUpdateSize-freeByteSize))
-}
-
-func EnableZeroFee() {
-	zeroFee = true
-}
-
-func DisableZeroFee() {
-	zeroFee = false
-}
-
-func IsZeroFee() bool {
-	return zeroFee
-}
-
-func NewZeroFee() *big.Int {
-	return big.NewInt(0)
-}
-
 func PayloadTxFee(payloadSize int) *big.Int {
 	if IsZeroFee() {
 		return NewZeroFee()
 	}
-	if payloadSize == 0 {
-		return new(big.Int).Set(baseTxAergo)
-	}
-	size := paymentDataSize(int64(payloadSize))
-	if size > payloadMaxSize {
-		size = payloadMaxSize
+
+	// set data fee
+	dataFee := paymentDataSize(int64(payloadSize))
+	if dataFee > payloadMaxSize {
+		dataFee = payloadMaxSize
 	}
-	return new(big.Int).Add(
-		baseTxAergo,
-		new(big.Int).Mul(
-			aerPerByte,
-			big.NewInt(size),
-		),
-	)
+	// return base fee + data fee
+	return new(big.Int).Add(baseTxAergo, CalcFee(aerPerByte, uint64(dataFee)))
 }
 
 func MaxPayloadTxFee(payloadSize int) *big.Int {
```

Comment:
`MaxPayloadTxFee` -> `MaxPayloadFee`

Reply:
it resolved in https://github.com/aergoio/aergo/commit/cc4ffbf860757323fe485c62d925fa26dc0a1248